### PR TITLE
[SPARK-27924][SQL][TEST][FOLLOW-UP] Enable Boolean-Predicate syntax tests

### PR DESCRIPTION
**Pull request by beliefer**
## What changes were proposed in this pull request?

This PR is a follow-up to https://github.com/apache/spark/pull/25074 

## How was this patch tested?

Pass the Jenkins with the newly update test files.

### CODE REVIEW

**Comment by beliefer** [comment]
@dongjoon-hyun Could you take a look at this PR?

File: sql/core/src/test/resources/sql-tests/results/pgSQL/boolean.sql.out [hunk 1]
```diff
@@ -566,176 +566,332 @@ struct<tf_12_ff_4:string,f1:boolean,f1:boolean>
...
 -- !query 61 output
-
+	true
+	true
+	true
 
 
 -- !query 62
-INSERT INTO BOOLTBL3 VALUES ('true', true, 1)
+SELECT '' AS "Not False", f1
```
Comment (wangyum) [inline sql/core/src/test/resources/sql-tests/results/pgSQL/boolean.sql.out]: `"Not False"` -> ``` `Not False` ```?

Comment (beliefer) [reply sql/core/src/test/resources/sql-tests/results/pgSQL/boolean.sql.out]: OK. I will change it. Thanks @wangyum .

**Comment by dongjoon-hyun** [comment]
Sure, @beliefer . I'll take a look.  Thank you for pinging me.

**Comment by dongjoon-hyun** [comment]
@beliefer . Please regenerate the result. That will recover the UT failures.

**Comment by dongjoon-hyun** [comment]
BTW, @beliefer . This had better be a followup of yours ([SPARK-27924][SQL] Support ANSI SQL Boolean-Predicate syntax). I updated the PR title. Also, I changed `How was this patch tested?` section. Since you updated the test cases which are verified by Jenkins. It should not be 'N/A'.

**Comment by dongjoon-hyun** [comment]
I made a PR to you, @beliefer . Please review and merge.
- https://github.com/beliefer/spark/pull/2

**Comment by maropu** [comment]
Does https://github.com/apache/spark/pull/25357 afffects the output in this pr? If so, I think we'd be better to merge that pr first to make code changes less.

**Comment by beliefer** [comment]
> BTW, @beliefer . This had better be a followup of yours ([SPARK-27924][SQL] Support ANSI SQL Boolean-Predicate syntax). I updated the PR title. Also, I changed `How was this patch tested?` section. Since you updated the test cases which are verified by Jenkins. It should not be 'N/A'.

Thank you very much!

**Comment by beliefer** [comment]
> I made a PR to you, @beliefer . Please review and merge.
> 
> * [beliefer#2](https://github.com/beliefer/spark/pull/2)

Thanks for your help! It's my mistake. I forget something.

**Comment by dongjoon-hyun** [comment]
@maropu . IMO, we need more review on #25357 ~ I'll comment on that PR.

**Review by dongjoon-hyun — APPROVED** [review]
@beliefer , @maropu , @wangyum .
I'll merge this first. 

+1, LGTM. Merged to master. Thank you, @beliefer .

**Comment by beliefer** [comment]
@dongjoon-hyun  Thanks for your help, review and work!

**Pull request closed without merging**